src/Omega_h_graph.hpp

```diff
@@ -29,8 +29,18 @@ struct Graph {
   LO nedges() const;
 };
 
+/** \brief combine the edges of two graphs that have the same set of vertices */
 Graph add_edges(Graph g1, Graph g2);
+/** \brief return a graph with the contents of b2c in the order specified by a2b */
```

Comment:
I would go with 'Traverse two graphs a2b and b2c to  form and return a graph from a2c'

Reply:
That makes sense to me.  Thank you.